kernel/src/aarch64/exceptions.rs

```diff
@@ -0,0 +1,133 @@
+use core::{
```

Comment:
I find this file and the accompanying.s file a little cryptic. Could you include a comment or two explaining what this does, maybe with a link to docs?